frontend/apps/marketing/src/components/list/ListContentfulDefinition.ts

```diff
@@ -0,0 +1,70 @@
+// Creates a definition for the FAQ Accordion component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+import {DEFAULT_ICON} from '@code-dot-org/component-library/list';
+
+import {componentSizeXSToLDefinition} from '@/components/common/definitions';
+
+export const ListContentfulComponentDefinition: ComponentDefinition = {
+  id: 'list',
+  name: 'Text List',
+  category: '03: Basic',
+  thumbnailUrl:
+    'https://images.ctfassets.net/90t6bu6vlf76/bkW5Mm9owpcZWnexKTRCq/0554f4990e4165ff8907f425bb501eee/component_list_thumbnail.png',
+  tooltip: {
+    description:
+      'A structured collection of list items in bullet-point format. Supports a custom icon.',
+    imageUrl:
+      'https://images.ctfassets.net/90t6bu6vlf76/5DICYZJBAh8MrBQQhkP5AO/836b3e29d0a036c3af7115c097b3c62f/component_list_tooltip.png',
+  },
+  // Adding an empty array here so no default style options show in the Design tab.
+  builtInStyles: [],
+  children: false,
+  variables: {
+    items: {
+      displayName: 'List Items',
+      type: 'Array',
+      group: 'content',
+      description:
+        'Accepts only the "List" content type entry that contain "List Item" entries',
+      validations: {
+        required: true,
+        bindingSourceType: ['entry'],
+      },
```

Comment:
Curious if we should add these to the FAQ and Carousel components too, @stephenliang and @levadadenys thoughts? I'm leaning yes.

Reply:
Yep that sounds good to me

Reply:
+1, thank you!